tools/pipelines/templates/include-publish-npm-package-steps.yml

```diff
@@ -145,6 +145,17 @@ steps:
               break
             fi
             rm publish_log
+            az rest --method patch --uri https://pkgs.dev.azure.com/{organization}/_apis/packaging/feeds/{feedId}/npm/$packageName/versions/{packageVersion}?api-version=7.1-preview.1 --body '{
```

Comment:
I don’t know what organization and feedId variables are for now, I’ll investigate more